tests/pyccel/test_pyccel.py

```diff
@@ -297,6 +324,13 @@ def test_imports_compile(language):
 #------------------------------------------------------------------------------
 @pytest.mark.xdist_incompatible
 def test_imports_in_folder(language):
+    if language == 'python':
+        current_folder = os.path.abspath(__file__)
+        result = current_folder.split('/')
+        result = result[:-1]
+        current_folder = "/".join(result)
+        if os.path.exists(current_folder+"/scripts/py/folder1") is False:
+            shutil.copytree(current_folder+"/scripts/folder1", current_folder+"/scripts/py/folder1")
```

Comment:
I am not sure this will work on windows. Please use functions from the `os.path` module